### PR TITLE
RES-112 Implement Sequence ID

### PR DESCRIPTION
**Pull request by SaganBolliger**
https://jira.numenta.com/browse/RES-112

@BoltzmannBrain Ready for review

### CODE REVIEW

File: htmresearch/frameworks/classification/classification_network.py [hunk 1]
```diff
@@ -204,6 +204,8 @@ def _linkRegions(network,
   network.link(previousRegionName, currentRegionName, "UniformLink", "")
   network.link(sensorRegionName, currentRegionName, "UniformLink", "",
                srcOutput="resetOut", destInput="resetIn")
+  network.link(sensorRegionName, currentRegionName, "UniformLink", "",
```
Comment (BoltzmannBrain) [inline htmresearch/frameworks/classification/classification_network.py]: Please add a comment in the docstring saying what this link does.

**Comment by BoltzmannBrain** [comment]
@SaganBolliger done w/ review.
The sequenceID input never gets connected to the classifier region. Linking the classifier region occurs [here](https://github.com/numenta/nupic.research/blob/master/htmresearch/frameworks/classification/classification_network.py#L329-337). Either add the sequenceID linking there, or have the classifier region link via `_linkRegions()` (preferred).

**Comment by BoltzmannBrain** [comment]
:+1: but please wait to merge until nupic #2726 is merged.

**Comment by SaganBolliger** [comment]
@BoltzmannBrain @scottpurdy I've implemented the change we discussed.

**Comment by BoltzmannBrain** [comment]
:+1:

**Comment by scottpurdy** [comment]
:+1:

**Pull request merged**